docs/getting-started/locally/add-instance.md

```diff
@@ -0,0 +1,43 @@
+# Add Instance
+
+Once you have a local instance running, you can edit it using the System Designer. The log messages indicate that your nio instance is available at `http://localhost:8181` and you need basic authentication to communicate with the instance. To connect to your local instance:
```

Comment:
it makes it sound like the log messages show you need BA, but that should be a separate sentence.

Reply:
Mention "access mode"